testing/matrix-sdk-test-utils/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+homepage = "https://github.com/matrix-org/matrix-rust-sdk"
+keywords = ["matrix", "chat", "messaging", "ruma"]
+license = "Apache-2.0"
+name = "matrix-sdk-test-utils"
+version = "0.1.0"
```

Let's keep it in sync with the rest of the workspace

```suggestion
version = "0.13.0"
```